### ISSUE TITLE
Is There a way to get a list of all mods allready on the crowdin without having to manually search

### ISSUE THREAD

**Issue by walksanatora** [IMB11/Loqui#2]

**Comment by IMB11** [comment]
Yes, see group_index.json on the translation repository: https://github.com/mineblock11/loqui-translations

**Comment by walksanatora** [comment]
if anyone else ask you can get a list of all mods with translations with this bash oneliner (requires jq)
`curl "https://raw.githubusercontent.com/mineblock11/loqui-translations/main/group_index.json" | jq 'keys[]' | sort`

**Comment by Nucceteere** [comment]
Thank you for the oneliner
